QtAwesome/iconic_font.py

```diff
@@ -1,22 +1,81 @@
 """Classes handling iconic fonts"""
 
 from __future__ import print_function
-from PyQt4.QtCore import Qt, QObject, QPoint, QRect, qRound, QByteArray
+from PyQt4.QtCore import Qt, QObject, QPoint, QRect, qRound, QByteArray, QTimer
 from PyQt4.QtGui import (QIcon, QColor, QIconEngine, QPainter, QPixmap,
                          QFontDatabase, QFont)
 import json
 import os
 
+
 _default_options = {
     'color': QColor(50, 50, 50),
     'scale_factor': 0.9,
 }
 
 
-class CharIconPainter:
+class CharIconAnimator:
```

Comment:
Yes, we can certainly do that, Do you want me to open a PR to your PR tonight to show what I mean?